### DISCUSSION TITLE
`getColumn` for `Table`

### DISCUSSION THREAD

**Discussion by scarf005** [rimbu-org/rimbu#217] [Ideas]
https://rimbu.org/api/rimbu/table/Table/interface#getrow

while it's possible to get list of column KV in row, afaik it's not possible to get list of row KV in column. it'd be useful in e.g implementing board games.

**Comment by vitoke** [comment]
Right, so let's say we have a a `.getColumn(col: C)` function, would this be the expected behavior?

```ts
const table = HashTableHashColumn.of(
  [1, "a", { a: 1 }],
  [2, "a", { a: 2 }],
  [2, "b", { a: 3 }]
);
const column = table.getColumn("a");
// column is a Map containing 2 entries: [1, { a: 1}] and [2, { a: 2 }]
```

Instead of `getColumn` it would probably be `streamColumn` so that you can choose in what kind of collection to put the result.

**Reply by scarf005** [reply]
hmm, i was thinking of `["a", "a", "b"]` for `table.getColumn(1)` (index) but maybe i misunderstood the table API
